homeassistant/components/nice_go/coordinator.py

```diff
@@ -164,6 +164,8 @@ async def _async_setup(self) -> None:
                 raise ConfigEntryAuthFailed from e
             except ApiError as e:
                 raise UpdateFailed from e
+            except (ConfigEntryAuthFailed, UpdateFailed):
+                raise
```

Comment:
```suggestion
```
I don't think the api will raise these exceptions as they are HA specific?

Reply:
The API doesn't raise these, yes, but they are raised here:

https://github.com/IceBotYT/core/blob/544e7c908fd8c8792f389b0ba973c717ff5f1bc4/homeassistant/components/nice_go/coordinator.py#L179-L184

Reply:
Sure, but there is no need to catch them to just raise them again.